File: src/turtle_control/scripts/turtle_control.py
import rospy
from geometry_msgs.msg import *

if __name__ == "__main__":
    rospy.init_node("turtle_control_py")
    pub = rospy.Publisher("/turtle1/cmd_vel", Twist, queue_size=10)

    pos = Twist()
    pos.linear.x = 1
    pos.angular.z = 1

    rate = rospy.Rate(1)
    while not rospy.is_shutdown():
        pub.publish(pos)
        rate.sleep()

File: src/turtle_control/src/turtle_control.cpp
#include "ros/ros.h"
#include "geometry_msgs/Twist.h"

using namespace std;

int main(int argc, char *argv[])
{
    ros::init(argc, argv, "turtle_control");
    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<geometry_msgs::Twist>("/turtle1/cmd_vel", 10);

    ros::Rate rate(1);

    geometry_msgs::Twist twist;
    twist.linear.x = 1;
    twist.angular.z = 1;

    while(ros::ok())
    {
        pub.publish(twist);
        rate.sleep();
    }
    
    return 0;
}
